src/main/java/com/okta/tools/authentication/BrowserAuthentication.java

```diff
@@ -65,7 +65,7 @@ public void start(final Stage stage) throws IOException {
 
         webEngine.getLoadWorker().stateProperty()
                 .addListener((ov, oldState, newState) -> {
-                    if (newState == State.SUCCEEDED) {
+                    if( webEngine.getDocument() != null ) {
```

Comment:
:art: the code style of this project uses:

```java
if (cond) {
   // stmts
}
```

Rather than:

```java
if( cond ) {
    // stmts
}
```